### ISSUE TITLE
Add Persian (Farsi) language

### ISSUE THREAD

**Issue by FaridAghili** [optimistdigital/nova-settings#47]
Translation for Persian (Farsi) language:

```json
// resources/lang/fa.php

{
  "novaSettings.navigationItemTitle": "تنظیمات",
  "novaSettings.saveButtonText": "ذخیره تنظیمات",
  "novaSettings.noSettingsFieldsText": "هیچ تنظیمی تعریف نشده است.",
  "novaSettings.settingsSuccessToast": "تنظیمات ذخیره شد.",
  "novaSettings.general": "عمومی"
}
```

**Comment by Tarpsvo** [comment]
Awesome, thanks!

**Comment by Tarpsvo** [comment]
Released in version 3.1.1. :)
